cypress/integration/unit_tests/applications.spec.ts

```diff
@@ -67,7 +67,7 @@ describe('Applications testing', () => {
     cy.runApplicationsTest('serviceMysqlBindWordpressPushApp');
   });
 
-  it('Push application with Github Source type and env vars and check it',  () => {
+  it.skip('Push application with Github Source type and env vars and check it',  () => {
```

Comment:
I think this is intended because I can read this in the PR description:
`Skipping Github Source test until release.`

Reply:
thanks @andreas-kupries for checking.
This is a test made to check the new Github Source Type feature made here: https://github.com/epinio/epinio-end-to-end-tests/issues/215
It will work once the latest release is out but until then it will fail.
I should have skipped it from begining. 
Since I am trying to fix now the tests after a month blind, I wanted to isolate as much as possible what works.
I will re-enable it after release as well as I will add new locator adjustments needed

Reply:
Ah. Thank you for explaining. I am then ok with having this in the PR.